features/expectations/uri.feature

```diff
@@ -1,15 +1,37 @@
-@stable
+@javascript @stable
 Feature: URI
 
-  Background:
+  Scenario: Matching HTTP message URI
     Given you expect HTTP message URI "/animals?type=cats"
-
-  Scenario: HTTP message URI match
     When real HTTP message URI is "/animals?type=cats"
     Then field "uri" is valid
     And Request or Response is valid
 
-  Scenario: Different HTTP message URI
+  Scenario: Non-matching HTTP message URI
+    Given you expect HTTP message URI "/animals?type=cats"
     When real HTTP message URI is "/animals?type=dogs"
     Then field "uri" is NOT valid
     And Request or Response is NOT valid
+
+  Scenario: URI with the same parameters in different order
+    Given you expect HTTP message URI "/animals?type=cats&count=2"
+    When real HTTP message URI is "/animals?count=2&type=cats"
+    Then field "uri" is valid
+    And Request or Response is valid
+
+  Scenario Outline: URI with parameter having multiple values
+    Given you expect HTTP message URI "/animals?type=cats&type=dogs"
+    When real HTTP message URI is <real-uri>
+    Then field "uri" <is-uri-valid>
+    And Request or Response <is-message-valid>
+
+    Examples:
+      | real-uri                       | is-uri-valid | is-message-valid |
+      | "/animals?type=cats&type=dogs" | is valid     | is valid         |
+      | "/animals?type=dogs&type=cats" | is NOT valid | is NOT valid     |
```

Comment:
Can we have just one column with is-valid? Splitting it between is-uri-valid and is-message seems redundant.

Reply:
True, I thought I wrote it in a single column 🤦‍♂. Will adjust.

Reply:
Fixed in e8d26b9.